editors/vscode/src/client.ts

```diff
@@ -487,6 +489,10 @@ function buildGoalsContent(goals : Goal[]) {
     footer =` </p>
 ￼       </body>
 ￼       </html>`;
+
+    // remove character sequences caused by multiline strings
+    header = header.replace(/[\u{0080}-\u{FFFF}]/gu,"");
+    footer = footer.replace(/[\u{0080}-\u{FFFF}]/gu,"");
```

Comment:
Interesting; indeed we should improve the document type that LP is sending to the client. I have some ideas here which could provide for very cool display [see for example what jsCoq goal buffer can do already] Let's hopefully discuss on a call.